Dockerfile

```diff
@@ -1,17 +1,17 @@
 FROM php:8.1-cli
-RUN apt-get update \
-    && apt-get install -y \
-        libzip-dev \
-        unzip \
-        git \
-        wget \
-    && docker-php-ext-install -j$(nproc) bcmath sockets \
-    && wget https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 \
-    && mv test-reporter-latest-linux-amd64 /usr/bin/cc-test-reporter  \
-    && chmod +x /usr/bin/cc-test-reporter \
-    && pecl install xdebug \
-    && docker-php-ext-enable xdebug && \
-    curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
+RUN apt-get update
+RUN apt-get install -y \
```

Comment:
why did this happen @exaby73 ? It will just bloat the cache layers of the container image. It is not considered good practice.